CHANGES.rst

```diff
@@ -52,7 +52,8 @@ Improvements
 Bugfixes
 ^^^^^^^^
 
-- Nothing so far
+- Prevent access to a badge design of a deleted category or an event (:issue:`5329`, :pr:`5334`, thanks
```

Comment:
I think this is something better suited for the 3.1.x branch. Also, let's not make lines even longer than all the lines around... so maybe wrap before the `:pr:` part.